_datascience/using_workflows.md

```diff
@@ -1,39 +1,235 @@
+- **[Nextflow Catalog](/datascience/nextflow_catalog)** - Fred Hutch curated Nextflow workflows
+- **[nf-core](https://nf-co.re/pipelines)** - Community-maintained Nextflow workflows (500+ pipelines)
+- **[GATK Workflows](https://github.com/gatk-workflows)** - Production genomics pipelines in WDL
+- **[Fred Hutch Nextflow repos](https://github.com/FredHutch?utf8=%E2%9C%93&q=nf+OR+nextflow&type=&language=)** - Community workflows
+
+### Execution Platforms
+
+- **[PROOF](/datascience/proof/)** - User-friendly interface for WDL workflows on Fred Hutch cluster
+- **[Gizmo](/scicomputing/compute_platforms/)** - Fred Hutch SLURM cluster for direct Nextflow execution
+- **Local execution** - Sprocket, miniWDL, or Nextflow on your workstation
+- **Cloud platforms** - AWS Batch, Terra, Cirro (via WDL or Nextflow)
+
+## Training and Learning Resources
+
+### WDL
+
+- **[Data Science Lab: Developing WDL Workflows](https://hutchdatascience.org/Developing_WDL_Workflows/)** - Comprehensive course
+- **[OpenWDL Learn WDL](https://github.com/openwdl/learn-wdl)** - Official tutorials
+- **[WDL Documentation](https://wdl-docs.readthedocs.io/)** - Language reference
+
+### Nextflow
+
+- **[Nextflow Training](https://training.nextflow.io/)** - Official Nextflow training
+- **[nf-core Usage Docs](https://nf-co.re/usage/introduction)** - Running nf-core pipelines
+- **[Nextflow Patterns](https://nextflow-io.github.io/patterns/)** - Common workflow patterns
+
+## Best Practices
+
+Regardless of which workflow system you choose:
+
+1. **Start small** - Test with small datasets before scaling up
+2. **Use version control** - Track your workflows in Git
+3. **Containerize** - Use Docker/Apptainer for reproducibility
+4. **Document** - Include README files and example inputs
+5. **Test** - Validate outputs before running large analyses
+6. **Share** - Contribute back to community libraries
+7. **Version workflows** - Tag releases when publishing results
 
-When performing bioinformatic analysis, scientists often need to perform a series of interconnected
-computational transformations of the raw input data. While it is possible to coordinate multiple tasks
-using BASH scripts or batch submission to a SLURM cluster, it can be far more convenient to use
-software which has been specifically designed to coordinate these types of workflows. Some examples
-of those workflow managers are:
+## Advanced Topics
 
-[Cromwell/WDL](https://cromwell.readthedocs.io/en/stable/), 
-and [Nextflow](https://nextflow.io/).
+Once comfortable with basic workflows:
 
+- **Cloud computing** - Scale to AWS Batch or other cloud platforms
+- **Custom containers** - Build your own Docker images with [WILDS Docker Library](https://github.com/getwilds/wilds-docker-library)
+- **Workflow optimization** - Fine-tune resource requests and parallelization
+- **CI/CD** - Automate workflow testing with GitHub Actions
+- **Publishing** - Share workflows via GitHub for reproducible research
```

Comment:
```suggestion
```
I would cut this out since there is no guidance offered on these topics and people will come across these organically.